__tests__/integrations.test.js

```diff
@@ -185,12 +185,22 @@ describe('Articles', () => {
           });
         });
     });
-    test('GET 404: Endpoint returns error message when article has no comments', () => {
+    test('GET 200: Endpoint returns an empty array when article has no comments', () => {
       return request(app)
         .get('/api/articles/2/comments')
-        .expect(404)
+        .expect(200)
+        .then(({ body: { comments } }) => {
+          const hasComments = comments.length;
+          expect(hasComments).toBeFalsy();
```

Comment:
Might have been more readable to just assert an empty array here.

Reply:
playing around with different matchers as don't want to just use the same ones all the time

Reply:
Fine to have a go, but if the end result is not as readable as a solution we have used before I would go with the readable option at all times. The goal is always good, readable code.